packages/payment-integration/src/test-utils/line-items.mock.ts

```diff
@@ -0,0 +1,94 @@
+import { DigitalItem, GiftCertificateItem, PhysicalItem } from "../cart/line-item";
```

Comment:
We should be using single quotes, but I believe it'll be once we apply `@bigcommerce/eslint-config` to the project.

Reply:
Yes agreed, I think this is my IDE doing things.

Reply:
This will move it as part of the `@bigcommerce/eslint-config` migration